src/components/PlaylistPage/MobileButtonsRow.js

```diff
@@ -2,13 +2,27 @@ import React, {PropTypes} from 'react';
 import Button from 'react-bootstrap/lib/Button';
 import Col from 'react-bootstrap/lib/Col';
 import Row from 'react-bootstrap/lib/Row';
+import Glyphicon from 'react-bootstrap/lib/Glyphicon';
```

Comment:
(Denne kommentaren gjelder for hele MobileButtonsRow). 
Se litt på kommentarene jeg har lagt inn på PR 239 og koordiner dette litt med Agnete. Ikke noe poeng å duplisere kode mellom FrontPage og PlaylistPage...